bin/cron/update_census_mapbox

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env ruby
+require_relative 'only_one'
+abort 'Script already running' unless only_one_running?(__FILE__)
+
+# The map on code.org/yourschool is run off of Mapbox.
+# This script updates the data on Mapbox by querying all rows from
+# census_summaries table for the appropriate school year,
+# creating maptiles with Mapbox's command line tool, Tippecanoe,
+# and uploading them to Mapbox's server.
+
+# After setting up authentication, provide the following CDO.* attributes:
+#   CDO.mapbox_upload_token: (the auth token for uploading tiles to Mapbox's servers)
+
+require_relative '../../dashboard/config/environment'
+require_relative '../../pegasus/helpers/properties'
+
+require_relative 'upload_to_mapbox'
+
+def make_location(lat, long)
+  return nil if lat.nil? || long.nil?
+
+  "#{format('%.6f', lat)}, #{format('%.6f', long)}"
+end
+
+def main
+  # Update summaries before updating the fusion table
+  # If any single teaches_cs category in a given year changes by more than
+  # 10% of the total number of schools then issue a Honeybadger notification.
+  threshold = School.count * 0.1
+  old_summary_counts = Census::CensusSummary.group(:school_year, :teaches_cs).count
+  Census::CensusSummary.summarize_census_data
+  new_summary_counts = Census::CensusSummary.group(:school_year, :teaches_cs).count
+
+  unless old_summary_counts.none? {|k, c| (c - new_summary_counts[k]).abs > threshold}
+    Honeybadger.notify(
+      error_message: "High rate of change in census_summaries. This may be expected if summary logic was recently changed or if a new data source was just added.",
+      error_class: "Census Summary High Rate of Change",
+      context: {
+        old_counts: old_summary_counts,
+        new_counts: new_summary_counts
+      }
+    )
+  end
+
+  #  The map should show the data from the school year that began
+  #  in the previous calendar year. So, Aug-Dec we show the previous school
+  #  year's data and then switch over to the current school year in Jan.
```

Comment:
Is this still true with using a constant school year? Do we want a constant school year or make it dynamic?